tests/api/tasks/has_encyclopedia_occurence.test.coffee

```diff
@@ -5,19 +5,90 @@ should = require 'should'
 { undesiredErr } = __.require 'apiTests', 'utils/utils'
 { checkEntities } = require '../utils/tasks'
 { createHuman, createWorkWithAuthor } = require '../fixtures/entities'
+{ getByUris } = require '../utils/entities'
 
 # Tests dependency: having a populated ElasticSearch wikidata index
 describe 'tasks:has-encyclopedia-occurence', ->
-  it 'should return true when author has work sourced in their wikipedia page', (done)->
-    createHuman { labels: { en: 'Victor Hugo' } }
+  it 'should return an empty object when author has no occurences', (done)->
+    humanLabel = 'Stanislas Lem' # has no homonyms
+    workLabel = 'fakeworklabel'
+    createHuman { labels: { en: humanLabel } }
     .then (human)->
-      createWorkWithAuthor human, 'Ruy Blas'
+      createWorkWithAuthor human, workLabel
       .then (work)-> checkEntities human.uri
       .then (tasks)->
         tasks.length.should.aboveOrEqual 1
-        Q535Task = tasks.find (task)-> task.suggestionUri is 'wd:Q535'
-        Q535Task.hasEncyclopediaOccurence.should.be.true()
+        task = tasks.find (task)-> task.suggestionUri
+        task.hasEncyclopediaOccurence.should.an.Object()
+        task.hasEncyclopediaOccurence.should.be.empty()
         done()
-      .catch undesiredErr(done)
+    .catch undesiredErr(done)
+
+    return
+
+  it 'should return an object of occurences uris when author has work sourced in their wikipedia page', (done)->
+    humanLabel = 'Stanislas Lem' # has no homonyms
+    workLabel = 'Solaris' # too short label to be automerged
+    createHuman { labels: { en: humanLabel } }
+    .then (human)->
+      createWorkWithAuthor human, workLabel
+      .then (work)-> checkEntities human.uri
+      .then (tasks)->
+        tasks.length.should.aboveOrEqual 1
+        task = tasks.find (task)-> task.suggestionUri.match /wd:/
+        task.hasEncyclopediaOccurence[0].uri.should.exist
```

Comment:
`task.hasEncyclopediaOccurence[0].uri.should.exist` is `undefined`
```suggestion
        task.hasEncyclopediaOccurence[0].uri.should.be.ok()
```

Reply:
hum, worked for me, but changed anyway

Reply:
the test does pass, it just doesn't test anything